pkg/httppublisher/publisher.go

```diff
@@ -0,0 +1,104 @@
+package httppublisher
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/canaryio/canary/pkg/sensor"
+)
+
+const MAX_BUFFER = 1000
+
+type Publisher struct {
+	url      string
+	interval time.Duration
+	buffer   []*sensor.Measurement
+	c        chan *sensor.Measurement
```

Comment:
I feel these could be named better than `c` or `i`. How about `measurementChan` and `measurementCount`. naming is indeed hard.